Configuration/Geometry/scripts/dict2023Geometry.py

```diff
@@ -353,6 +353,78 @@
             'from Geometry.EcalMapping.EcalMappingRecord_cfi import *',
         ],
         "era" : "run2_HE_2017, run2_HF_2017, run2_HCAL_2017, run3_HB, phase2_hcal, phase2_hgcal",
+    },
+    "C3" : {

```

Comment:
@kpedro88 
shouldn't edits in this file be accompanied by more changes?
(e.g. README, generated geometry cff files and definition of new workflows)